<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teleoperation</class>
 <widget class="QWidget" name="Teleoperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-580</x>
     <y>-10</y>
     <width>1780</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#bf40b8;&quot;&gt;唯品会&lt;/span&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:18pt; color:#eeeeec;&quot;&gt;无人车配送后台监控系统&lt;/span&gt;&lt;span style=&quot; font-size:20pt; color:#eeeeec;&quot;&gt;   --&lt;/span&gt;&lt;span style=&quot; color:#eeeeec;&quot;&gt;遥操作&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_forward">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>642</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>前进</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_backward">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>679</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_stop">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>679</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_turnRight">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>642</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>右转</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>679</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_horn">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>679</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>鸣笛</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_light">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>642</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>开灯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_turnLeft">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>642</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>左转</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_taskAsk">
   <property name="geometry">
    <rect>
     <x>521</x>
     <y>679</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>任务查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_adas">
   <property name="geometry">
    <rect>
     <x>521</x>
     <y>642</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>辅助驾驶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_boxSelect">
   <property name="geometry">
    <rect>
     <x>688</x>
     <y>642</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>货柜选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_vehicleInfo">
   <property name="geometry">
    <rect>
     <x>688</x>
     <y>679</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>车辆信息</string>
   </property>
  </widget>
  <widget class="QWebView" name="webView_main" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>48</y>
     <width>781</width>
     <height>541</height>
    </rect>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QWebView" name="webView_right" native="true">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>50</y>
     <width>240</width>
     <height>160</height>
    </rect>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QPushButton" name="button_changeViewport_left">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>切换为主视角</string>
   </property>
  </widget>
  <widget class="QWebView" name="webView_front" native="true">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>240</y>
     <width>240</width>
     <height>160</height>
    </rect>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QPushButton" name="button_changeViewport_back">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>切换为主视角</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_changeViewport_map">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>600</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>切换为主视角</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_changeViewport_right">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>210</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>切换为主视角</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_changeViewport_front">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>400</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>切换为主视角</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>600</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#008cec;&quot;&gt;主视角&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#008cec;&quot;&gt;左相机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#008cec;&quot;&gt;后相机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#008cec;&quot;&gt;实时路线&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>400</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#008cec;&quot;&gt;前相机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#008cec;&quot;&gt;右相机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>600</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QWebView" name="webView_back" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>240</width>
     <height>160</height>
    </rect>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QWebView" name="webView_map" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>240</width>
     <height>160</height>
    </rect>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QWebView" name="webView_left" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>240</width>
     <height>160</height>
    </rect>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_leftAR">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_rightAR">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>210</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_frontAR">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_backAR">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
